msrezaie/city-scrapers-newark
# Build spider: Newark Board of Education

## What's this PR do?
This PR adds a new spider for Board of Education, Newark, NJ.

## Why are we doing this?
Built a new spider as per request.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl newnj_nbe -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid.

## Are there any smells or added technical debt to note?
No smell or technical debt, just one note about the location of the meetings. Many meetings for this organization takes place virtually, so the relevant links for joining these meetings have been saved in the `links` field.
